src/Card/index.js

```diff
@@ -0,0 +1,109 @@
+// @flow
+import * as React from "react";
+import styled from "styled-components";
+import defaultTokens from "@kiwicom/orbit-design-tokens";
+import { withTheme } from "theming";
+
+import { CloseCircle } from "../icons";
+import Button from "../Button";
+import Heading from "../Heading";
+import Text from "../Text";
+
+type Props = {
+  title: string,
+  description: string,
+  closable: boolean,
+  sectioned: boolean,
+  icon?: React.ComponentType<*>,
+  children?: React.Node,
+  onClose: () => void,
+  theme: typeof defaultTokens,
+};
+
+const StyledCard = styled.div`
+  width: 100%;
+  height: 100%;
+  background: ${props => props.theme.backgroundCard};
+  border-radius: ${props => props.theme.borderRadiusNormal};
+  border-width: ${props => props.theme.borderWidthCard};
+  border-style: ${props => props.theme.borderStyleCard};
+  border-color: ${props => props.theme.borderColorCard};
+  box-sizing: border-box;
+  overflow: hidden;
+`;
+
+const StyledCardHeader = styled.div`
+  position: relative;
+  width: 100%;
+  padding: ${props => props.theme.spaceLarge};
+  padding-right: ${props => (props.closable ? props.theme.spaceXXLarge : props.theme.spaceLarge)};
+  box-sizing: border-box;
+`;
+
+const StyledCardBody = styled.div`
+  width: 100%;
+  padding: ${props =>
```

Comment:
this function is not too readable, would be nicer to define it as normal named function and avoid nested ternary operator

Reply:
Fixed.